packages/main/src/CustomListItem.js

```diff
@@ -23,6 +23,18 @@ const metadata = {
 		},
 	},
 	properties: /** @lends sap.ui.webcomponents.main.CustomListItem.prototype */ {
+		/**
+		 * Defines the text alternative of the <code>ui5-li</code>.
+		 * If not provided a default text alternative will be set, if present.
```

Comment:
<b>Note:</b> If not provided a default text alternative will be set, if present.